identify/extensions.py

```diff
@@ -246,18 +253,24 @@
     'BUILD': {'text', 'bazel'},
     'BUILD.bazel': {'text', 'bazel'},
     'CMakeLists.txt': EXTENSIONS['cmake'],
+    'CHANGELOG': EXTENSIONS['txt'],
+    'CODE_OF_CONDUCT': EXTENSIONS['txt'],
```

Comment:
iirc github requires this to be `.md` or `.rst` -- is there conventions of this elsewhere?

Reply:
Ah, wasn't sure about that. Not that I could find, so I dropped it.